egs/zeroth_korean/s5/RESULTS

```diff
@@ -0,0 +1,63 @@
+#!/bin/bash
```

Comment:
Please add a file s5/README.txt that explains something about the data: what type of data, how much of it, how you can obtain it, what the license is, things like that.

Reply:
I added README.txt